pkg/format/Dockerfile

```diff
@@ -6,17 +6,27 @@ RUN apk add --no-cache --initdb -p /out \
     busybox \
     e2fsprogs \
     e2fsprogs-extra \
-    jq \
+    btrfs-progs \
+    xfsprogs \
     musl \
     sfdisk \
+    util-linux \
     && true
 RUN rm -rf /out/etc/apk /out/lib/apk /out/var/cache
 
+FROM linuxkit/alpine:9bcf61f605ef0ce36cc94d59b8eac307862de6e1 AS build
```

Comment:
And this uses a different alpine base to the other Dockerfiles. It would be good if a PR updates them somewhere that it would update the base in the other dockerfiles it touches as well